README.md

```diff
@@ -1,6 +1,12 @@
-# Node.js
-
-[![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/nodejs/node?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) [![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/29/badge)](https://bestpractices.coreinfrastructure.org/projects/29)
+<p align="center">
+  <a href="https://nodejs.org/">
+    <img src="https://nodejs.org/static/images/logo-light.svg" width="400" />
```

Comment:
I'd say we should follow guidance of things like http://webaim.org/techniques/alttext/#logos. "Node.js home page" is probably what we want per that doc. Specifically, the `alt` text should likely not contain the word "logo".